### PR TITLE
V1 blog post

### PR DESCRIPTION
**Pull request by flavio**
Add blog post covering the v1 release. Fixes https://github.com/kubewarden/kubewarden.io/issues/103

This is currently a draft release, it's blocked by the final v1 release.

**TODO before publishing:**
* [x] update the publication date inside of the article metadata

### CODE REVIEW

File: content/blog/2022/06/v1-release.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+title: Kubewarden reaches 1.0.0 release 🎉
+authors:
+- Flavio Castelli
+date: 2022-06-08
+---
+
+Today, a year and a half since its conception, we're thrilled to announce
+Kubewarden's 1.0.0 release! 🎊 🥳
+
+Let's take a quick look at the major achievements of the project to see what `v1`
```
[the remark this answers is not in the document]
Comment (flavio) [reply content/blog/2022/06/v1-release.md:11]: What do you mean? :sweat_smile: Do you have any ideas (git pushes, releases, lines of code,...)

[the remark this answers is not in the document]
Comment (divya-mohan0209) [reply content/blog/2022/06/v1-release.md:11]: Haha, not specific to lines of code. I meant the number of features like X going into alpha, Y going into beta or something like that?

File: content/blog/2022/06/v1-release.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+## Writing policies
+
+Kubewarden v1 allows Policy Authors to write policies using either Go, Rust or Swift.
+Each language features a rich SDK that simplifies the process of validating and
+mutating Kubernetes admission requests.
+
+Creating a new policy can be done quickly by scaffolding one of our official
+policy templates projects. New policies created in this way will feature best
+practices such as: code linters, unit tests, end to end tests, policy signing
+and publishing. All of that automated using GitHub Actions.
```
[the remark this answers is not in the document]
Comment (divya-mohan0209) [reply content/blog/2022/06/v1-release.md:23]: Okay, how about?
```suggestion
and publishing. All of that is automated using GitHub Actions.
```

**Review by jvanz — APPROVED** [review]

**Review by raulcabello — APPROVED** [review]
it looks great! Thanks!

File: content/blog/2022/06/v1-release.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (flavio) [reply content/blog/2022/06/v1-release.md:23]: That's great, thanks!

**Comment by jvanz** [comment]
@flavio The `v1.0.0-rc1` is released. Do you want to publish this blog post now or only after the final `v1.0.0`?

File: content/blog/2022/06/v1-release.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+Today, a year and a half since its conception, we're thrilled to announce
+the release of Kubewarden v1.0.0! 🎊 🥳
+
+Let's take a quick look at the major achievements of the project to see what `v1`
+brings both to Policy Authors and Operators.
+
+## Writing policies
+
+Kubewarden v1 allows Policy Authors to write policies using either Go, Rust, or Swift.
```
Comment (olblak) [inline content/blog/2022/06/v1-release.md:16]:
```suggestion
Kubewarden v1 allows Policy Authors to write policies using any programming language that can generate a WebAssembly module. Such as using Go, Rust, or Swift. More information is available [here](https://webassembly.org/getting-started/developers-guide/). 
```

I think, it's an important element to mention

Comment (flavio) [reply content/blog/2022/06/v1-release.md:16]: This is true, but we have SDKs only for these languages

**Comment by divya-mohan0209** [comment]
A release blog, IMO, should be published on the day OR the day after the release is cut.

File: content/blog/2022/06/v1-release.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
+---
+title: Kubewarden reaches 1.0.0 release 🎉
+authors:
+- Flavio Castelli
+date: 2022-06-22
+---
+
+Kubewarden is a policy engine for Kubernetes that is part of CNCF Sandobox.
```
Comment (viccuad) [inline content/blog/2022/06/v1-release.md:8]:
```suggestion
Kubewarden is a policy engine for Kubernetes that is part of CNCF Sandbox.
```

Comment (divya-mohan0209) [reply content/blog/2022/06/v1-release.md]: I would go so far as to change it and break it into two separate sentences as below:
```suggestion
Kubewarden is a policy engine for Kubernetes. The project is a part of [CNCF Sandbox](https://www.kubewarden.io/blog/2022/06/cncf-sandbox-inclusion/).
```

**Comment by kkaempf** [comment]
I'd like to see the initial paragraph be less like a "wall of text". Can we just list what sets Kubewarden apart in a bullet list ?
Like

- makes policies shareable, first-class objects
- stores policies in container registries
- simplifies the learning curve by offering well-known programming languages
- compiles to WebAssembly to make policies portable
- provides a secure software supply chain

(+ highlight the keywords as bold for example)

File: content/blog/2022/06/v1-release.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+What makes Kubewarden different from other similar projects is the way
+policies are written and distributed.
+
+Kubewarden policies can be written both using traditional programming languages
+(like Go or Rust) and with Domain Specific Languages (like Rego).
+Policies are compiled into [WebAssembly](https://webassembly.org)
+modules that are distributed via regular container registries.
+
+Kubewarden policies are compiled once and, thanks to the flexibility of WebAssembly,
```
Comment (divya-mohan0209) [inline content/blog/2022/06/v1-release.md:17]:
```suggestion
Once compiled, Kubewarden policies can then be run on any kind of operating system and architecture
```

File: content/blog/2022/06/v1-release.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+can then be run on any kind of operating system and architecture that is supported
```
Comment (divya-mohan0209) [inline content/blog/2022/06/v1-release.md:18]:
```suggestion
that is supported by the Kubewarden stack owing to the portability of the WebAssembly framework.
```

File: content/blog/2022/06/v1-release.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+This allows both Policy Authors and Operators to have a consistent observability
+stack both for Cloud Native applications and Kubernetes policies.
+
+## Software Secure Supply Chains
+
+The topic of [Software Secure Supply Chains](https://www.nist.gov/itl/executive-order-14028-improving-nations-cybersecurity/software-security-supply-chains)
+has been gaining momentum during 2021 and has become a high priority topic for
+many organizations.
+
+Kubewarden wants to help Operators to address the challenges posed by Secure
```
Comment (divya-mohan0209) [inline content/blog/2022/06/v1-release.md:98]:
```suggestion
Kubewarden wants to help Operators to address the challenges faced during implementation of Secure
```

File: content/blog/2022/06/v1-release.md [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+Supply Chains initiatives.
```
Comment (divya-mohan0209) [inline content/blog/2022/06/v1-release.md:99]:
```suggestion
Supply Chain initiatives.
```

File: content/blog/2022/06/v1-release.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+
+To do that, Kubewarden leverages the [Sigstore](https://www.sigstore.dev/)
+project to implement software signing and verification.
```
Comment (divya-mohan0209) [inline content/blog/2022/06/v1-release.md:102]:
```suggestion
project while implementing software signing and verification.
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
A few comments, hopefully they're useful

**Pull request merged**